Add note about VS to the .NET Framework ref assemblies article (#34600)
dotnet/docs: docs/framework/migration-guide/reference-assemblies.md

  01. Run _msbuild /t:restore_.

- If your project is an SDK-style project, you don't need to do anything. The NuGet restore action is automatically run when the project is built.

> > [!IMPORTANT]
> > While using reference assemblies makes it possible to build the project on the command line, it still doesn't allow loading it in Visual Studio.
> To continue building apps that target unsupported versions of .NET Framework in Visual Studio, the only workaround is to use an older version of
> Visual Studio.

Reviewer: ```suggestion
> [!IMPORTANT]
> While using reference assemblies makes it possible to build the project from the command line, it still doesn't allow loading it in Visual Studio. To continue building apps that target unsupported versions of .NET Framework in Visual Studio, the only workaround is to use an older version of Visual Studio.
```

Reviewer 2: ```suggestion
> [!IMPORTANT]
> Using reference assemblies makes it possible to build projects that target unsupported versions of .NET Framework from the command line. However, you still can't load these projects in newer versions of Visual Studio. To continue building these apps in Visual Studio, the only workaround is to use an older version of Visual Studio.
```

Author: Committed, thank you!
